pages/evm/update-price-feeds.mdx

```diff
@@ -8,10 +8,16 @@ import { Tab, Tabs } from "nextra-theme-docs";
 
 # Update Price Feeds
 
-**updatePriceFeeds(updateData: bytes[])**
+Update the on-chain price feeds using the provided `updateData`, which contains serialized and signed price update data from Pyth Network.
+You can retrieve the latest price `updateData` for a given set of price feeds from the price service API.
```

Comment:
should we link to the price service API whenever its mentioned? imo makes the UX better

Reply:
Yeah, once they're ready. I figure it doesn't make sense to document the current price service since it'll be replaced by hermes.

Reply:
makes sense 👍🏻